apps/passport-client/components/screens/VerifyScreen.tsx

```diff
@@ -89,9 +93,7 @@ export function VerifyScreen() {
         <Spacer h={48} />
         <Placeholder minH={160}>
           {result?.valid === false && <TextCenter>{result.message}</TextCenter>}
-          {result?.valid === true && (
-            <CardElem expanded card={getCard(result)} />
-          )}
+          {result && getCard(result)}
```

Comment:
this will throw an error if `result.valid === false`, making the message above redundant